day-4/contact_tracker/src/App.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import ReactDOM from 'react-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import Input from './components/Input.js';
+import 'mui-react/dist/mui-react.css';
+//import RaisedButton from 'material-ui/RaisedButton';
+
+import TableComponent from './components/Table.js';
+
+var key =0;
+var _this;
```

Comment:
let's move away from _this and start using arrow functions. google ES6 arrow functions if you need to, but you guys are definitely far enough along to be switching to full ES6 syntax